sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -19,6 +19,29 @@
     secondaryLabel="Previous section"
     class="mx-0"
   >
+    <template #primary-buttons v-if="isLastPage">
```

Comment:
The save draft is currently being visible in the read-only view. The reason being `notDraft` variable is not set by the parent to the component and other 2 variables `isFirstPage` and `processing` is false.

![image](https://github.com/user-attachments/assets/f4f5b4be-b504-439d-a3c6-9149496f0b5b)